### ISSUE TITLE
Is there a seed for fine-tuning?

### ISSUE THREAD

**Issue by simplelifetime** [haotian-liu/LLaVA#823]
### Question

I'm wondering if there is a seed for fine-tuning. Currently the reproduced results for second stage fine-tuning are all different from the results of official checkpoint. I'm certain that the training script is exactly the same as provided in this repo. I'm wondering if setting the different random seed can cause such problem.

**Comment by liyang-7** [comment]
Same problem.  The loss is different even with fixed seed in TrainingArguments. 
May I ask if you have a method to set a fixed random seed to maintain consistency in the training process?

**Comment by Akhalee** [comment]
How do you fix them?

**Comment by xirui-li** [comment]
Same problem
